blocks/layout-grid/src/index.js

```diff
@@ -32,7 +32,7 @@ function getColumnAttributes( total, breakpoints ) {
 export function registerBlock() {
 	registerBlockType( 'jetpack/layout-grid', {
 		title: __( 'Layout Grid', 'layout-grid' ),
-		description: __( 'Align blocks to to a global grid, with support for responsive breakpoints.', 'layout-grid' ),
```

Comment:
Well spotted!

Reply:
Props @donalirl in https://github.com/Automattic/wpcom-blocks/issues/169#issue-597041838!